src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -255,7 +255,7 @@ public async Task<Document> ApplyCodeFixesForSpecificDiagnosticIdAsync(Document
             var fixAllService = document.Project.Solution.Workspace.Services.GetRequiredService<IFixAllGetFixesService>();
 
             var solution = await fixAllService.GetFixAllChangedSolutionAsync(
-                fixCollection.FixAllState.CreateFixAllContext(progressTracker, cancellationToken)).ConfigureAwait(false);
+                new FixAllContext(fixCollection.FixAllState, progressTracker, cancellationToken)).ConfigureAwait(false);
```

Comment:
why this change?

Reply:
`FixAllState` had to be moved to the shared layer, but `CreateFixAllContext` could not be moved as it accessed other internal types not available in shared layer. That method was anyways just a single object creation, so it has just been inlined here.